crates/easter/src/cover.rs

```diff
@@ -13,15 +15,21 @@ pub enum Error {
 
 impl Display for Error {
     fn fmt(&self, fmt: &mut Formatter) -> fmt::Result {
-        match self {
-            &Error::InvalidAssignTarget(_) => {
-                fmt.write_str("invalid assignment pattern")
-            }
-            &Error::InvalidPropPatt(_) => {
-                fmt.write_str("invalid object property in assignment pattern")
-            }
+        fmt.write_str(self.description())
+    }
+}
+
+impl error::Error for Error {
```

Comment:
I'd suggest to just use already imported `impl StdError` here and in other places.